src/js/components/Header/PlayingBar.react.js

```diff
@@ -29,7 +29,7 @@ export default class PlayingBar extends Component {
         queue: React.PropTypes.array,
         queueCursor: React.PropTypes.number,
         shuffle: React.PropTypes.bool,
-        repeat: React.PropTypes.string
+        repeat: React.PropTypes.bool
```

Comment:
He changed it because of [this](https://files.gitter.im/KeitIG/museeks/A4uk/blob)